src/blkio.rs

```diff
@@ -432,6 +436,16 @@ impl BlkIoController {
     /// Constructs a new `BlkIoController` with `root` serving as the root of the control group.
     pub fn new(root: PathBuf, v2: bool) -> Self {
         Self {
+            mount_root: Default::default(),
```

Comment:
BTW, I found the base field wasn't used, was it better to use base field as the cgroup's mount root path?

Reply:
@c3d  @lifupan  Thank you both for the review, i find that it's true that `base` is not used. I'll modify and push a new version later.